src/main/java/com/iemr/flw/controller/BeneficiaryOTPGatewayController.java

```diff
@@ -0,0 +1,115 @@
+package com.iemr.flw.controller;
+
+import com.iemr.flw.domain.iemr.OtpBeneficiary;
+import com.iemr.flw.dto.iemr.OTPRequestParsor;
+import com.iemr.flw.dto.iemr.OtpRequestDTO;
+import com.iemr.flw.mapper.InputMapper;
+import com.iemr.flw.service.OTPHandler;
+import com.iemr.flw.utils.response.OutputResponse;
+import io.lettuce.core.dynamic.annotation.Param;
+import io.swagger.v3.oas.annotations.Operation;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.ws.rs.core.MediaType;
+
+@RestController
+@RequestMapping("/beneficiary")
+public class BeneficiaryOTPGatewayController {
+    final Logger logger = LoggerFactory.getLogger(this.getClass().getName());
+
+    @Autowired
+    private OTPHandler otpHandler;
+
+    @Operation(summary = "Send OTP")
+    @RequestMapping(value = "/sendOTP", method = RequestMethod.POST, produces = MediaType.APPLICATION_JSON,headers = "Authorization")
```

Comment:
How is this API going to be protected from attacks?
I can potentially trigger n number of requests for a number and exhaust our SMS balance.

@srinuatpiramal @sac2kadam There should be rate limiting for this API.
@madhavaramu83 please ticket this separately.